.github/workflows/reusable-release_gssutils.yaml

```diff
@@ -0,0 +1,14 @@
+name: Trigger a workflow at gssutils that updates csvcubed package
```

Comment:
Might be nice to make this name a bit shorter, perhaps `Trigger csvcubed update in gss-utils`?